demos/http/http_demo_plaintext/demo_config.h

```diff
@@ -0,0 +1,107 @@
+/*
+ * Copyright (C) 2020 Amazon.com, Inc. or its affiliates.  All Rights Reserved.
+ *
+ * Permission is hereby granted, free of charge, to any person obtaining a copy of
+ * this software and associated documentation files (the "Software"), to deal in
+ * the Software without restriction, including without limitation the rights to
+ * use, copy, modify, merge, publish, distribute, sublicense, and/or sell copies of
+ * the Software, and to permit persons to whom the Software is furnished to do so,
+ * subject to the following conditions:
+ *
+ * The above copyright notice and this permission notice shall be included in all
+ * copies or substantial portions of the Software.
+ *
+ * THE SOFTWARE IS PROVIDED "AS IS", WITHOUT WARRANTY OF ANY KIND, EXPRESS OR
+ * IMPLIED, INCLUDING BUT NOT LIMITED TO THE WARRANTIES OF MERCHANTABILITY, FITNESS
+ * FOR A PARTICULAR PURPOSE AND NONINFRINGEMENT. IN NO EVENT SHALL THE AUTHORS OR
+ * COPYRIGHT HOLDERS BE LIABLE FOR ANY CLAIM, DAMAGES OR OTHER LIABILITY, WHETHER
+ * IN AN ACTION OF CONTRACT, TORT OR OTHERWISE, ARISING FROM, OUT OF OR IN
+ * CONNECTION WITH THE SOFTWARE OR THE USE OR OTHER DEALINGS IN THE SOFTWARE.
+ */
+
+#ifndef DEMO_CONFIG_H_
+#define DEMO_CONFIG_H_
+
+/**************************************************/
+/******* DO NOT CHANGE the following order ********/
+/**************************************************/
+
+/* Logging related header files are required to be included in the following order:
+ * 1. Include the header file "logging_levels.h".
+ * 2. Define LIBRARY_LOG_NAME and  LIBRARY_LOG_LEVEL.
+ * 3. Include the header file "logging_stack.h".
+ */
+
+/* Include header that defines log levels. */
+#include "logging_levels.h"
+
+/* Logging configuration for the Demo. */
+#define LIBRARY_LOG_NAME     "DEMO"
+#define LIBRARY_LOG_LEVEL    LOG_INFO
+
+#include "logging_stack.h"
+
+/************ End of logging configuration ****************/
+
+/**
+ * @brief HTTP server host name.
+ *
+ * This demo uses httpbin.org: A simple HTTP Request & Response Service.
+ */
+#define SERVER_HOST    "httpbin.org"
```

Comment:
Why does the test depend on this service?  Who is it operated by, what is the SLA, are they okay with you pointing a lot of developers (and automated testers) at it with this code, etc.

Why not use a local server?  It would avoid all of these issues.

Reply:
This service is operated by [postman](postman.com). I was unable to find any data on the SLA or availability of [httpbin.org](httpbin.org). However, we have used this service in the past for AFR demos. Another option would be [postman-echo.com](postman-echo.com). 

We can use a local server. That would be nice because we would be able to determine how many people are running our demos and even where they come from. However, there is added time and cost to setting this up. What is your opinion on this? Should we setup our own server on EC2?

Reply:
Update: After discussing with @gkwicker, we are having the user run a local http server before running the demos: `python3 tools/http-echo-server/http_server.py`